lib/engine/game/g_1893/buy_eva_from_other_player.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module BuyEvaFromOtherPlayer
+  def buy_eva_from_other_player(entity, company, price)
```

Comment:
why do you need this module? why don't just override process_buy_company

Reply:
Hm.... I think my plan was to use it from buy_sell_par_shares.rb in 1893 as well, but that does not seem to need it? Right, need to look into my solution again to see if I can do as you recommend.